its/src/test/java/com/sonar/it/scanner/msbuild/ScannerMSBuildTest.java

```diff
@@ -29,10 +29,7 @@
 import java.io.IOException;
 import java.nio.file.LinkOption;
 import java.nio.file.Path;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```

Comment:
I know that in the company we/they don't like using `*` in imports for java.

Reply:
Ok. That was changed automatically by IntelliJ.